sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java

```diff
@@ -444,6 +444,13 @@ public EventDirectorySteps(
           webDriverHelpers.selectFromCombobox(
               FILTER_BY_RISK_LEVEL, RiskLevelValues.getCaptionForName(riskLevel));
         });
+    When(
+        "I select random German Risk level filter among the filter options from API",
```

Comment:
"filter options from API"

please adjust the method name to reflect that we select the risk level based on the value set in the Event created by API